eng/Workarounds.targets

```diff
@@ -25,6 +29,20 @@
     <PackageReference Include="Internal.AspNetCore.BuildTasks" PrivateAssets="All" Version="$(InternalAspNetCoreBuildTasksPackageVersion)" IsImplicitlyDefined="true" />
   </ItemGroup>
 
+  <ItemGroup>
+    <!-- Workaround when there is no vNext SDK available, copy known apphost/framework reference info from 3.0 -->
+    <KnownAppHostPack 
+      Include="@(KnownAppHostPack->WithMetadataValue('TargetFramework', 'netcoreapp3.0'))"
+      TargetFramework="$(DefaultNetCoreTargetFramework)"
+      Condition="(@(KnownAppHostPack->Count()) != '0') AND (!(@(KnownAppHostPack->AnyHaveMetadataValue('TargetFramework', '$(DefaultNetCoreTargetFramework)'))))"
```

Comment:
Why the extra layer of parentheses around both of the AND'ed clauses?

Reply:
Paranoia 😆